code/obj/item/clothing/hats.dm

```diff
@@ -1961,6 +1961,117 @@ TYPEINFO(/obj/item/clothing/head/lesbian_hat)
 	item_state = "space_replica"
 	desc = "A replica of an old space helmet. Looks spaceworthy regardless."
 
+/obj/item/clothing/head/giraffehat
+	name = "giraffe hat"
+	desc = "Great for finally reaching those tender tree-top leaves."
+	icon = 'icons/obj/large/32x48.dmi'
+	icon_state = "giraffehat"
+	item_state = "giraffehat"
+
+/obj/item/clothing/head/rhinobeetle
+	name = "rhino beetle helm"
+	desc = "A lightweight helm styled like a rhinocerous beetle's horn. Not sturdy enough for you to flip your enemies over with, sadly."
+	icon_state = "rhinobeetle"
+	item_state = "rhinobeetle"
+
+/obj/item/clothing/head/stagbeetle
+	name = "stag beetle helm"
+	desc = "A lightweight helm styled like a stag beetle's mandibles. Not actually a functional set of grabbers, unfortunately."
+	icon_state = "stagbeetle"
+	item_state = "stagbeetle"
+
+ABSTRACT_TYPE(/obj/item/clothing/head/elephanthat)
+/obj/item/clothing/head/elephanthat
+	name = "elephant hat"
+	desc = "Quite cozy, if you don't mind the trunk smacking you in the face when you walk."
+	icon_state = "elephant-gold"
+	item_state = "elephant-gold"
+	var/hatcolor = "default"
```

Comment:
`hatcolor` only appears to have use as a var scoped within the random type as temp storage for the icon states, no need to have it on the obj